user-service/controller/user-controller.js

```diff
@@ -44,4 +48,62 @@ export async function loginUser(req, res) {
     } catch (err) {
         return res.status(500).json({ message: `Database failure when logging in as ${username}!` })
     }
-}
\ No newline at end of file
+}
+
+export async function forgetPassword(req, res) {
+    try {
+        const { username } = req.body;
+        // Check the existance of input

```

Comment:
nit: the comment is unnecessary since we have the message explanation below

Reply:
ok will make the change.